datahub-web-react/src/app/previewV2/EntityHeader.tsx

```diff
@@ -94,17 +94,28 @@ const EntityHeader: React.FC<EntityHeaderProps> = ({
 }) => {
     const linkProps = useEmbeddedProfileLinkProps();
 
+    // Handle click on the link - run custom onClick before navigation
+    const handleLinkClick = React.useCallback(
```

Comment:
The previous behavior straight up would not work. The link was intercepting the clicks.